app/scripts/controllers/provisionedService.js

```diff
@@ -0,0 +1,190 @@
+'use strict';
+
+/**
+ * @ngdoc function
+ * @name openshiftConsole.controller:ProvisionedServiceController
+ * @description
+ * Controller of the openshiftConsole
+ */
+angular.module('openshiftConsole')
+  .controller('ProvisionedServiceController', function ($scope,
+                                             $routeParams,
+                                             DataService,
+                                             ProjectsService,
+                                             $filter) {
+    $scope.projectName = $routeParams.project;
+    $scope.displayName = null;
+    $scope.serviceInstance = null;
+    $scope.serviceInstances = null;
+    $scope.serviceClasses = {};
+    $scope.alerts = {};
+    //$scope.renderOptions = $scope.renderOptions || {};
+    //$scope.renderOptions.hideFilterWidget = true;
+
+    $scope.breadcrumbs = [
+      {
+        title: "Provisioned Services",
+        link: "project/" + $routeParams.project + "/browse/provisioned"
+      },
+      {
+        title: $routeParams.service
+      }
+    ];
+
+    $scope.podFailureReasons = {
+      "Pending": "This pod will not receive traffic until all of its containers have been created."
+    };
```

Comment:
Looks like you have a fair amount of code from the service controller that is not needed here.